gemrb/plugins/WEDImporter/WEDImporter.cpp

```diff
@@ -36,21 +38,14 @@
 
 using namespace GemRB;
 
-struct wed_polygon {
-	ieDword FirstVertex;
-	ieDword CountVertex;
-	ieWord Flags; // two bytes in the original: type and height with height currently unused
-	ieWord MinX, MaxX, MinY, MaxY;
-};
-
 //the net sizeof(wed_polygon) is 0x12 but not all compilers know that
 #define WED_POLYGON_SIZE  0x12
 
 WEDImporter::WEDImporter(void)
 {
 	str = NULL;
 	OverlaysCount = DoorsCount = OverlaysOffset = SecHeaderOffset = 0;
-	DoorPolygonsCount = DoorsOffset = DoorTilesOffset = PILTOffset = 0;
+	DoorPolygonsCount = DoorsOffset = DoorTilesOffset = PLTOffset = 0;
```

Comment:
Codacy found an issue: [Variable 'PLTOffset' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)